src/rsz/test/cpp/TestBufferRemoval.cc

```diff
@@ -226,21 +226,25 @@ TEST_F(BufRemTest, SlackImproves)
 
   resizer_->initBlock();
   db_->setLogger(&logger_);
-  resizer_->journalBeginTest();
-  resizer_->logger()->setDebugLevel(utl::RSZ, "journal", 1);
 
-  auto insts = std::make_unique<sta::InstanceSeq>();
-  odb::dbInst* inst1 = block->findInst("b2");
-  sta::Instance* sta_inst1 = db_network_->dbToSta(inst1);
-  insts->emplace_back(sta_inst1);
+  {
+    IncrementalParasiticsGuard guard(resizer_);
```

Comment:
Should this guard be in removeBuffers?  It seems odd to add it to a test that previously did no invalidation and suggests it is a client responsibilty.